bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRCommand.java

```diff
@@ -33,6 +33,8 @@
 
 public class CSRCommand implements CommandHandler {
     private static final String CSR_LABEL = "csr";
+    private static final String CSR_NEEDED_MARKER = "<!-- csr: 'needed' -->";
+    private static final String CSR_UNNEEDED_MARKER = "<!-- csr: 'unneeded' -->";
```

Comment:
Did you intend to look for this in the CSRBot?

Reply:
I don't understand what you mean. You set these markers, but I don't see anything looking for them.

Reply:
```java
// file CheckWorkItem.java
-    private final Pattern metadataComments = Pattern.compile("<!-- (?:(add|remove) (?:contributor|reviewer))|(?:summary: ')|(?:solves: ')|(?:additional required reviewers)|(?:jep: ')");
+    private final Pattern metadataComments = Pattern.compile("<!-- (?:(add|remove) (?:contributor|reviewer))|(?:summary: ')|(?:solves: ')|(?:additional required reviewers)|(?:jep: ')|(?:csr: ')");
```

The newly added `(?:csr: ')` can search it and update the metadata so that the check can be re-run.

Reply:
Ah of course, thanks.